src/app/content/practiceQuestions/reducer.ts

```diff
@@ -24,11 +24,11 @@ const reducer: Reducer<State, AnyAction> = (state = initialState, action): State
       const shouldBeOpen = action.payload.query[modalQueryParameterName] === modalUrlName
         && action.payload.action === 'PUSH';
 
-      return {...state, open: shouldBeOpen, selectedSection: null, questions: [], questionAnswers: {}};
+      return { ...state, open: shouldBeOpen, selectedSection: null, questions: [], questionAnswers: {} };
```

Comment:
You probably have some kind of extensions installed that is fixing these brackets automatically
Please disable it or configure it to use our lint config - this will be giving a lot of diff noise in large files and is making review harder

Reply:
Format document command in VSC does it. I will try to configure it, that it will meet the project requirements.